assets/scss/main.css.scss

```diff
@@ -15,6 +15,8 @@
 @forward "uswds";
 @forward "footer";
 
+$block-background-color: #fafafa;
```

Comment:
I see this is copied from the design system styles, but I might plan to surface this as it's strange we don't use one of our standard design system tokens (like a `base-lightest` or something).